### ISSUE TITLE
Emojis not showing in Linux

### ISSUE THREAD

**Issue by gpeto91** [contentco/quill-emoji#3]
I implemented this plugin initially in Windows and it worked perfectly. But when I access my project in Linux, the emojis doesn't load correctly. Do I have to install anything additional for it to work in Linux? I know that Linux may not have support for the font you are using natively.  

Here is how it's showing me in Linux:

![issue_quill_emoji](https://cloud.githubusercontent.com/assets/8106116/25496962/5bbabc24-2b59-11e7-9eb9-d99044fa73a4.png)

**Edit**: For what I see, the font is user dependent and I think that is a big problem. I tried to run my project in another PC with Windows 7 and it didn't load the emojis as well. Just in Firefox, but not in Chrome.

**Comment by himynameistimli** [comment]
I think this could be something to help with adding support for linux: http://probablement.net/txt/emojilinux

Recommends three things we can do:
1) In the very short term, installing the Symbola font by default.
2) Then, developing a system-wide implementation of either colour fonts, SVG fonts, or characters substitution with SVG images, using one the existing free emoji sets.
3) Eventually, making our own emoji sets!

What would be the the most effective way to get it to work (more) universally but not add too much bloat to the overall package.

**Comment by Wessek** [comment]
Hello himynameistimli and contentco,

Thank you very much for your response!

I ran into the same issue, and the bottom-line here is: no developers can really afford to publish a web application that relies on the user's system configuration to work (especially for these emoji fonts). We could have, potentially, thousands of users using a webapp and reporting us these bugs.

We'll be working to implement this solution, which is great, as you described in steps 2 and 3; if we succeed, we'll provide you our piece of code. :-)
